<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>191</width>
     <height>176</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Frequency</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_freq"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Amplitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_amp"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Cycles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_cycles"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Averages</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_avg"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Tx Gain</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_txgain"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Rx Gain</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_rxgain"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>601</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabSignal">
    <attribute name="title">
     <string>Signal</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabData">
    <attribute name="title">
     <string>Data</string>
    </attribute>
    <widget class="QTableWidget" name="tableData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>571</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_Status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>750</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Status: Not Connected...</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>279</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButtonRun">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Data Label</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditDataLabel"/>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonRecord">
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="checkBoxFilter">
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxSweep">
      <property name="text">
       <string>Sweep</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
